src/pages/ProductList/components/GoodList/Heart.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+import { AiOutlineHeart } from 'react-icons/ai';
+import { AiFillHeart } from 'react-icons/ai';
+
+export const Heart = () => {
+  const [liked, setLiked] = useState(false);
```

Comment:
```suggestion
const [isLiked, setIsLiked] = useState(false)
```
boolean type은 의문문 형태로 짓는 편 입니다.

Reply:
반영했습니다.